p2p/__init__.py

```diff
@@ -515,12 +515,28 @@ def junk_content_item(self, slug):
             'content_item_state_code': 'junk'
         })
 
+    def content_item_exists(self, slug):
+        """
+        Checks for the existance of a slug in content services
+        """
+        exists = False
+        try:
+            _ = self.get("/content_items/%s/exists" % (slug))
+        except P2PNotFound:
+            exists = False
+        except ValueError:
+            # The return from this function is not valid json,
```

Comment:
P2P Replies with the string "Content item exists" instead of returning valid json.